wasm-node/javascript/demo/demo.mjs

```diff
@@ -19,7 +19,7 @@
 
 import * as smoldot from '../dist/mjs/index-nodejs.js';
 import { WebSocketServer } from 'ws';
-import * as process from 'node:process';
+import process from 'node:process';
```

Comment:
Note that without this change I'm getting `process.on is not a function` errors. Have no idea why.